apps/dashboard/src/main.ts

```diff
@@ -81,7 +81,6 @@ app.component('ToggleButton', ToggleButton);
 app.component('Steps', Steps);
 app.component('Calendar', Calendar);
 app.component('ConfirmDialog', ConfirmDialog);
-app.component('InputSwitch', InputSwitch);
```

Comment:
Why delete this? We still use input switches for boolean form inputs

Reply:
its already there